docs/sources/plugins/developing/development.md

```diff
@@ -123,9 +120,9 @@ If a panel receives data and hooks into the `data-received` event then it should
 We currently have three different examples that you can fork/download to get started developing your grafana plugin.
```

Comment:
```suggestion
We have three different examples that you can fork/download to get started developing your Grafana plugin.
```

Reply:
@marcusolsson and @mckn , I suspect you will want to add or replace this section around the May timeframe.